packages/api-kit/tests/e2e/addSafeDelegate.test.ts

```diff
@@ -11,17 +13,21 @@ const PRIVATE_KEY_1 = '0x83a415ca62e11f5fa5567e98450d0f82ae19ff36ef876c10a8d448c
 const PRIVATE_KEY_2 = '0xb0057716d5917badaf911b193b12b910811c1497b5bada8d7711f758981c3773'
 
 let safeApiKit: SafeApiKit
-let signer: Signer
+let signer: AddSafeDelegateProps['signer']
 
 describe('addSafeDelegate', () => {
   before(async () => {
     safeApiKit = getApiKit('https://safe-transaction-sepolia.staging.5afe.dev/api')
-    signer = new ethers.Wallet(PRIVATE_KEY_1)
+    signer = createWalletClient({
+      chain: sepolia,
+      transport: http(),
+      account: privateKeyToAccount(PRIVATE_KEY_1)
+    })
   })
 
   it('should fail if Label is empty', async () => {
     const delegateAddress = '0x9cCBDE03eDd71074ea9c49e413FA9CDfF16D263B'
-    const delegatorAddress = await signer.getAddress()
+    const delegatorAddress = signer.account!.address
```

Comment:
This can be undefined right? Why is better than something like:

```
const accounts = await walletClient.getAddresses() 
```

Reply:
Yes that might be better. Updated